.nvmrc

```diff
@@ -1 +0,0 @@
-12
```

Comment:
this should probably be restored and set to `lts/*`

Reply:
done